db/db_impl/db_impl.cc

```diff
@@ -4701,8 +4701,10 @@ Status DBImpl::CreateColumnFamilyWithImport(
 
   import_job.Cleanup(status);
   if (!status.ok()) {
-    DropColumnFamily(*handle);
-    DestroyColumnFamilyHandle(*handle);
+    // TODO: Should handle this error or log this?
+    DropColumnFamily(*handle).PermitUncheckedError();
```

Comment:
I think we should handle the error here, and also check `versions_->io_status` if necessary because this method potentially writes to MANIFEST.

Reply:
If I add ```status = DropColumnFamily(*handle)```, then it will overwrite the previous errored status and new error will be returned(instead of returning previous error).  Shouldn't we log the error instead as we want to drop and delete the column handle after status is not ok?

Reply:
I am thinking about using another `tmp_status` object, instead of assigning to `status`.

Reply:
Logging the new error SGTM